subxt/src/storage/storage_address.rs

```diff
@@ -229,7 +229,7 @@ pub fn make_static_storage_map_key<T: codec::Encode>(t: T) -> StaticStorageMapKe
 }
 
 /// Construct a new dynamic storage lookup to the root of some entry.
-pub fn dynamic_root(
+pub fn dynamic_iter(
```

Comment:
Ah, but now the `dynamic_iter` doesn't actually provide the ability to iterate over subsets of keys like the static one does.

Maybe we should just remove this function entirely, since people can just use `dynamic` below with an empty vec to achieve the same, or with a subset of keys to hopefully be able to iterate whatever they like.

Reply:
(and if we remove this, remember to update the guide :)